.gitmodules

```diff
@@ -36,6 +33,9 @@
 [submodule "externals/Catch"]
 	path = externals/Catch
 	url = https://github.com/philsquared/Catch.git
+[submodule "externals/assimp"]
+	path = externals/assimp
+	url = https://github.com/Chainsawkitten/assimp.git
```

Comment:
Use shallow or else there isn't much point in using our own assimp repository (it'll still download entire tree).